### DISCUSSION TITLE
TransformImage in ImageMagick v7

### DISCUSSION THREAD

**Discussion by FrankS70** [ImageMagick/ImageMagick#4725] [Help]
Hi,

I'm using TransformImage from MagickCore. When I try to migrate my source from using v6 to v7, TransformImage is no more existing (?) resp compiler will not find anymore.

I looked for porting notes, but all I found was that a 4th parameter for exception is added.

Where is my issue? Or how can I replace TransformImage?

I have installed ImageMagick 7.1 on Debian bullseye with backports.

Thanks
Frank

**Comment by snibgo** [comment]
See the recent question https://github.com/ImageMagick/ImageMagick/discussions/4707
